dev/integration_tests/flutter_gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -344,9 +344,13 @@
 			baseConfigurationReference = 7AFA3C8E1D35360C0083082E /* Release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEVELOPMENT_TEAM = S8QB4VV633;
```

Comment:
I'm guessing we don't want to check this in?

Reply:
yeah I'll fix this.